src/v/cluster/topics_frontend.cc

```diff
@@ -827,6 +827,119 @@ ss::future<std::error_code> topics_frontend::force_update_partition_replicas(
       _stm, _features, _as, std::move(cmd), tout, term);
 }
 
+ss::future<result<topics_frontend::force_move_plan>>
+topics_frontend::force_replicas_from_nodes(
```

Comment:
What do you think about making this method part of partition balancer backend ? This way we could use all information available in planer f.e. node disk sizes.